tests/experimental/metric_repository/test_column_descriptive_metrics_metric_retriever.py

```diff
@@ -216,7 +230,166 @@ def test_get_metrics_metrics_missing():
             metric_name="column.min",
             column="col1",
             value=None,
-            exception=MetricException(type="TBD", message="TBD"),
+            exception=MetricException(
+                type="Not found",
+                message="Metric was not successfully computed but exception was not found.",
+            ),
```

Comment:
would this be / could this be "exception was not raised?" Not sure what the correct syntax here would be.

Reply:
after discussion: happy to keep as is. We don't know if the missing exception was because of correct or incorrect behavior around raising Exceptions (?)